### PR TITLE
Add community version controlnet performance testing

### PR DESCRIPTION
**Pull request by ccssu**
- model: sd_xl_base_1.0.safetensors
- batch size: 1
- image size: 1024x1024
- steps: 20

Timings for 20 steps at 1024x1024
| e2e                    | Basic  | Lora   | ControlNet |
| ---------------------- | ------ | ------ | ---------- |
| PyTorch                | 5.57 s | 5.62 s | 7.95 s     |
| OneDiff                | 3.16 s | 3.16 s | 5.45 s     |
| Percentage improvement | 43%    | 44%    | 31%        |

Image throughput for 20 steps at 1024x1024
| it/s                   | Basic    | Lora     | ControlNet |
| ---------------------- | -------- | -------- | ---------- |
| PyTorch                | 4.04it/s | 3.99it/s | 2.87it/s   |
| OneDiff                | 7.61it/s | 7.51it/s | 4.40it/s   |
| Percentage improvement | 88%      | 88%      | 53%        |

- onediff: d387e7e9 (origin/main, origin/HEAD, main) 
- oneflow: version: 0.9.1.dev20240120+cu121 git_commit: 287b806

### CODE REVIEW

File: onediff_comfy_nodes/README.md [hunk 1]
```diff
@@ -6,27 +6,29 @@
...
 
-Updated on DEC 7, 2023. Device: RTX 3090
+Updated on January 23, 2024. Device: RTX 3090
 
-| SDXL1.0-base (1024x1024)                                       | torch(Baseline) | onediff(Optimized) | Percentage improvement |
+<!-- | SDXL1.0-base (1024x1024)                                       | torch(Baseline) | onediff(Optimized) | Percentage improvement |
 | -------------------------------------------------------------- | --------------- | ------------------ | ---------------------- |
 | [Stable Diffusion workflow(UNet)](workflows/model-speedup.png) | 4.08it/s        | 6.70it/s           | 64.2 %                 |
-| [LoRA workflow](workflows/model-speedup-lora.png)              | 4.05it/s        | 6.69it/s           | 65.1 %                 |
+| [LoRA workflow](workflows/model-speedup-lora.png)              | 4.05it/s        | 6.69it/s           | 65.1 %                 | -->
```
Comment (strint) [inline onediff_comfy_nodes/README.md:14]: 为什么把性能数据注释掉了

**Comment by strint** [comment]
性能数据都加到 readme 吧

File: onediff_comfy_nodes/README.md (hunk 1 above)
Comment (ccssu) [reply onediff_comfy_nodes/README.md:14]: 计划用图片直观展示
image

Comment (ccssu) [reply onediff_comfy_nodes/README.md:14]: 图 2.
image

Comment (ccssu) [reply onediff_comfy_nodes/README.md:14]: > 为什么把性能数据注释掉了

是用图片 还是 直接用 表格？

Comment (strint) [reply onediff_comfy_nodes/README.md:14]: 赞，那就用图片吧，比较清楚

Comment (strint) [reply onediff_comfy_nodes/README.md:14]: 后面如果需要展现很多设备的数据时，再附加表格。

**Review by strint — APPROVED** [review]

**Pull request merged**